app/test/big_blob_test.go

```diff
@@ -54,40 +53,6 @@ func (s *BigBlobSuite) SetupSuite() {
 	require.NoError(t, cctx.WaitForNextBlock())
 }
 
-// TestErrBlobsTooLarge verifies that submitting a ~1.9 MiB blob hits ErrBlobsTooLarge.
-func (s *BigBlobSuite) TestErrBlobsTooLarge() {
-	t := s.T()
-
-	type testCase struct {
-		name string
-		blob *share.Blob
-		// want is the expected tx response ABCI code.
-		want uint32
-	}
-	testCases := []testCase{
-		{
-			name: "~ 1.9 MiB blob",
-			blob: newBlobWithSize(2_000_000),
-			want: blobtypes.ErrTotalBlobSizeTooLarge.ABCICode(),
-		},
-	}
-
-	txClient, err := testnode.NewTxClientFromContext(s.cctx)
-	require.NoError(t, err)
-
-	for _, tc := range testCases {
-		s.Run(tc.name, func() {
-			subCtx, cancel := context.WithTimeout(s.cctx.GoContext(), 30*time.Second)
-			defer cancel()
-			res, err := txClient.SubmitPayForBlob(subCtx, []*share.Blob{tc.blob}, user.SetGasLimitAndGasPrice(1e9, appconsts.DefaultMinGasPrice))
-			require.Error(t, err)
-			require.Nil(t, res)
-			code := err.(*user.BroadcastTxError).Code
-			require.Equal(t, tc.want, code, err.Error())
-		})
-	}
-}
```

Comment:
why remove this test?

don't we want to keep ensuring that too large of txs cannot be submitted?

Reply:
`ErrTotalBlobSizeTooLarge ` isn't thrown in v4. It was thrown by the ante handler that was removed in this PR but that ante handler only applies to app version v1.

Reply:
`ErrTxExceedsMaxSize` is checked by another test csae in this suite.

Reply:
I see, thanks for clarifying!